projects/npm-tools/CONTRIBUTING.md

```diff
@@ -121,3 +122,18 @@ yarn publish --tag beta
 For bonus points, create a draft PR like [this one](https://github.com/liferay/liferay-npm-tools/pull/201) so that others have visibility into what you are doing. After the release is done you should feel free to close the PR and delete the temporary branch that you pushed (but keep the tag in case anybody ever wants to look up the source that was published to NPM).
 
 Finally, visit the versions tab on [the NPM registry page](https://www.npmjs.com/package/@liferay/npm-scripts) to confirm that your release is visible and is appropriately tagged as "beta" (not "latest").
+
+## Integrating in [liferay-portal](https://github.com/liferay/liferay-portal)
+
+One of the design goals for and motivating features of `@liferay/npm-scripts` is to abstract over our development dependencies by "hiding" them inside a single package. This means that updating liferay-portal is basically just a single `yarn add` command from inside the `modules/` directory:
+
+1. `yarn add -W --dev @liferay/npm-scripts@a.b.c`, where `a.b.c` is the version that you wish to integrate.
+2. Check the `yarn.lock` file for unwanted duplication (see below for some notes on this).
+3. Perform testing and sanity-checking (the exact nature of this depends on what changes are included in the release).
+
+If you detect unwanted duplication in the `yarn.lock`, there are a number of tools you can try (from least to most aggressive):
+
+1. `npx yarn-deduplicate yarn.lock` (historically this one used to be enough).
+2. `yarn install` (at the time of writing, this one is often required).
```

Comment:
Maybe we should say here that you must use portal's yarn version (which currently is v10). Otherwise very strange errors happen.

Reply:
Let me see if I can come up with a succinct way of saying that.

Reply:
Added https://github.com/liferay/liferay-frontend-projects/pull/239/commits/8b0d7099c506c7e8d117551625fa3f72eb371dd8

The temptation to mention `portool` was strong, but I resisted... 💪

Reply:
:clap: